lib/src/server/handler.dart

```diff
@@ -31,32 +31,33 @@ import 'service.dart';
 /// Handles an incoming gRPC call.
 class ServerHandler_ extends ServiceCall {
   final ServerTransportStream _stream;
-  final Service Function(String service) _serviceLookup;
+  final Service? Function(String service) _serviceLookup;
   final List<Interceptor> _interceptors;
-  final CodecRegistry _codecRegistry;
+  final CodecRegistry? _codecRegistry;
 
-  StreamSubscription<GrpcMessage> _incomingSubscription;
+  // ignore: cancel_subscriptions
+  StreamSubscription<GrpcMessage>? _incomingSubscription;
 
-  Service _service;
-  ServiceMethod _descriptor;
+  Service? _service;
+  ServiceMethod? _descriptor;
```

Comment:
I think both of these should become `late` non-nullable given how they are used.

Reply:
Done